strapi/patches/@strapi+plugin-graphql+4.3.4.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/node_modules/@strapi/plugin-graphql/server/services/builders/entity.js b/node_modules/@strapi/plugin-graphql/server/services/builders/entity.js
```

Comment:
please remove this from the PR before merging - if it makes sense to have it, we can do that in a separate PR